notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>one question, are you optimizing over precision at k? bellow you show the results of precision at k</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/687/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes, the script optimizes the primary metric, which is precision@k.
I should add some description to explain how this is done.